CollAction/Startup.cs

```diff
@@ -285,7 +287,14 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseAuthentication();
 
-            app.UseStaticFiles();
+            app.UseStaticFiles(new StaticFileOptions()
+            {
+                OnPrepareResponse = ctx =>
```

Comment:
This will cache all static resources, including images, fonts and style sheets. But you only hash the JavaScript.

Reply:
The stylesheets are, as far as I can see, packed into the javascript. So they're included.

The images are changed for a few reasons:
1 Resizing to optimize performance
2 Actually changing the image

For 1, I would suggest that it's fine to keep caching the old version for a week. 2 is rare, and only happens on redesign, I would suggest we do a rename if that happens.